core/src/main/java/org/apache/iceberg/util/SortOrderUtil.java

```diff
@@ -44,7 +44,11 @@ public static SortOrder buildSortOrder(Table table) {
     return buildSortOrder(table.schema(), table.spec(), table.sortOrder());
   }
 
-  public static SortOrder buildSortOrder(Schema schema, PartitionSpec spec, SortOrder sortOrder) {
+  public static SortOrder buildSortOrder(Table table, SortOrder sortOrder) {
```

Comment:
I had to introduce this method to stay on a single line in a few places.

Reply:
Small nit here, but I would add just a note that this builds a sort order using both the table partition spec and the user supplied sort order.

Reply:
Added a comment.